### ISSUE TITLE
Refine main page donate error state

### ISSUE THREAD

**Issue by youriwims** [mozilla/donate-wagtail#336]
### Issue: [Elements must have sufficient color contrast](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=AxeChrome)

**Element path:** `#payments__braintree-errors-paypal`

**Snippet:** `<div id="payments__braintree-errors-paypal" class="payments__braintree-error">Payment cancelled</div>`
<img width="735" alt="Screen Shot 2019-10-01 at 12 58 32 PM" src="https://user-images.githubusercontent.com/36484608/65983333-43a3d000-e44b-11e9-823f-5ac248214d97.png">

How to fix:
Fix any of the following:
- Increase contrast ratio by updating error `background-color`

====

More Info:

Comments: The main concern here is that the background-color isn't easily/quickly discernible as an 'error'. Related to #305 
Resources: https://webaim.org/resources/contrastchecker/?fcolor=FFE6E8&bcolor=FFFFFF

**Comment by nicklee** [comment]
@youriwims / @solarissmoke any idea how I can replicate this error?

**Comment by solarissmoke** [comment]
@nicklee start a paypal payment, and then close the paypal popup window.

**Comment by nicklee** [comment]
I've moved the error above the form - users could easily miss it on smaller browser / monitor sizes. And updated the text color to black.

**Comment by nicklee** [comment]
![Screenshot 2019-10-03 at 12 28 28](https://user-images.githubusercontent.com/298766/66123271-7d0a4600-e5d9-11e9-9aaa-e9aeab6383a3.png)

**Comment by youriwims** [comment]
Perfect. https://webaim.org/resources/contrastchecker/?fcolor=000000&bcolor=FFE6E8
